RUFAS/biophysical/manure/storage/composting.py

```diff
@@ -0,0 +1,553 @@
+        Passive windrow.
+    STATIC_PILE: str
+        Static pile.
+
+    """
+
+    INTENSIVE_WINDROW = "intensive windrow"
+    PASSIVE_WINDROW = "passive windrow"
+    STATIC_PILE = "static pile"
+
+
+ACHIEVABLE_METHANE_EMISSION: float = 0.24
+"""Achievable emission of methane from dairy manure (m^3 methane / kg volatile solids)."""
+
+MCF_COMPOSTING_STATIC_PILE: float = 0.005
+"""The MCF for static pile composting."""
+
+MCF_LOWER_BOUND_TEMPERATURE: float = 15.0
+"""The lower bound temperature for determining MCF for windrow composting."""
+
+MCF_UPPER_BOUND_TEMPERATURE: float = 25.0
+"""The upper bound temperature for determining MCF for windrow composting."""
+
+MCF_COMPOSTING_WINDROW_LOW: float = 0.005
+"""The MCF for windrow composting when the air temperature is below the lower bound temperature."""
+
+MCF_COMPOSTING_WINDROW_MEDIUM: float = 0.01
+"""The MCF for windrow composting when the air temperature is between the lower and upper bound temperature."""
+
+MCF_COMPOSTING_WINDROW_HIGH: float = 0.015
+"""The MCF for windrow composting when the air temperature is above the upper bound temperature."""
+
+DEFAULT_CARBON_FRACTION_AVAILABLE_IN_VSD: float = 0.5
+"""Default carbon content (percent by mass) of manure degradable volatile solids (unitless, [0, 1])."""
+
+DEFAULT_CARBON_FRACTION_AVAILABLE_IN_VSND: float = 0.35
+"""Default carbon content (percent by mass) of manure non-degradable volatile solids (unitless, [0, 1])."""
+
+DEFAULT_EFFECT_OF_MOISTURE_ON_MICROBIAL_DECOMPOSITION: float = 0.65
+"""The default effect of moisture on microbial decomposition."""
+
+FIRST_ORDER_DECAYING_COEFFICIENT: float = 0.1
+"""The first order decaying coefficient."""
+
+DEFAULT_LAG_TIME: int = 2
+"""Default lag time used in the calculation of the carbon decomposition rate (days). Default is set to 2."""
+
+DEFAULT_DAYS_SINCE_LAST_TURNING: int = 1
+"""Default days since the previous compost turning event (days). Default is set to 1."""
+
+DEFAULT_MOLE_FRACTION_OF_OXYGEN: float = 0.15
+"""The default mole fraction of oxygen in the air wihtin the compost layer."""
+
+OXYGEN_HALF_SATURATION_CONSTANT: float = 0.02
+"""The half saturation constant of Oxygen gas (O2)"""
+
+AMBIENT_AIR_MOLE_FRACTION_OF_OXYGEN: float = 0.21
+"""The mole fraction of oxygen in ambient air."""
+
+FRACTION_NITROGEN_LOST_TO_AMMONIA_EMISSION: dict[CompostingType, float] = {
+    CompostingType.STATIC_PILE: 0.5,
+    CompostingType.PASSIVE_WINDROW: 0.45,
+    CompostingType.INTENSIVE_WINDROW: 0.5,
+}
+
+FRACTION_NITROGEN_LOST_TO_DIRECT_N2O_EMISSION: dict[CompostingType, float] = {
+    CompostingType.STATIC_PILE: 0.06,
+    CompostingType.PASSIVE_WINDROW: 0.04,
+    CompostingType.INTENSIVE_WINDROW: 0.06,
+}
+
+FRACTION_NITROGEN_LOST_TO_LEACHING: dict[CompostingType, float] = {
+    CompostingType.STATIC_PILE: 0.06,
+    CompostingType.PASSIVE_WINDROW: 0.04,
+    CompostingType.INTENSIVE_WINDROW: 0.06,
+}
+
+EFFECTIVENESS_OF_MICROBIAL_DECOMPOSITION_RATE: float = 2.73e-3
```

Comment:
```suggestion
DEFAULT_EFFECTIVENESS_OF_MICROBIAL_DECOMPOSITION_RATE: float = 2.37e-3
```
This is an error from the original module. I know we are trying not to make functional changes in the refresh, but, in the solid manure methods, we are going to be changing the outcome of C decomposition and DM loss due to needing to use degradable and non-degradable VS instead of manure and bedding mass for determining C available. The output values are going to be different regardless, so, might as well fix these errors too instead of putting them off til post-refresh! The C decomposition calculations need to be revisited anyway so I will make a spreadsheet of manual calculations to utilize in checking refreshed module outputs. 
![image](https://github.com/user-attachments/assets/e7e89bc7-547f-4c06-9ade-06db6b103867)

Reply:
implemented rate change but confirmed with Elle that the DEFAULT should not be part of the name.